### ISSUE TITLE
Periodically check the set custom wallpaper for changes for integration with 3rd party apps

### ISSUE THREAD

**Issue by Shtoomik** [rocksdanister/lively#967]
**Is your feature request related to a problem? Please describe.**
I use Lively (mostly rain effect) together with Displayfusion. 

- Displayfusion is changing the wallpaper randomly at an interval and I've set Lively to the aggregated dual screen wallpaper as a custom background.
- Lively does not change the wallpaper when Displayfusion changes it/ generates a new verion (same name, same location).
- I can go into the Lively settings and change to another wallpaper and back to the Displayfusion wallpaper to have it pick up the new version. 

**Describe the solution you'd like**
Periodic check in Lively (by imagesize, checksum,..) if the set wallpaper has changed and if yes set Lively to the new one - _Transition would be awesome_

**Describe alternatives you've considered**
#137 functionality to change wallpapers in Lively directly, but I have multiple folders with curated resolutions for different screen sizes and Displayfusion options and other 3rd party apps are probably (in the near future) more powerful and customizable then integrated ones.

**Additional context**
Thanks for the awesome app

**Comment by rocksdanister** [comment]
I am bit confused.. you want "Rain v2" wallpaper to update with the picture of desktop wallpaper as DisplayFusion changes it?

**Comment by Shtoomik** [comment]
Yes, Displayfusion uses _\AppData\Local\DisplayFusion\Wallpaper_1.png_ as background, If the file gets changed I would like to have Lively to update the cached wallpaper under _\AppData\Local\Lively Wallpaper\Library\wallpapers\aahrtq0t.dme\wallpapers_.

I've also realized I have to delete the cached wallpaper in _\AppData\Local\Lively Wallpaper\Library\wallpapers\aahrtq0t.dme\wallpapers_ to choose a new one with the same name (and location) otherwise it will be added with a (1).

The current Windows wallpaper seems to be set in the registry in the 
HKEY_USERS\S-1-5-21-SID\Control Panel\Desktop\Wallpaper key - this could be used to set current Windows wallpaper in general, with a checkmark in the Lively wallpaper settings (?)

![Screenshot 2021-12-31 115903](https://user-images.githubusercontent.com/3796212/147820185-b9742e66-fe54-4acd-bb78-a94cb41f4f66.png)

**Comment by rocksdanister** [comment]
>I've also realized I have to delete the cached wallpaper in \AppData\Local\Lively Wallpaper\Library\wallpapers\aahrtq0t.dme\wallpapers to choose a new one with the same name (and location) otherwise it will be added with a (1).

This is by design.

>Yes, Displayfusion uses \AppData\Local\DisplayFusion\Wallpaper_1.png as background, If the file gets changed I would like to have Lively to update the cached wallpaper under \AppData\Local\Lively Wallpaper\Library\wallpapers\aahrtq0t.dme\wallpapers.

This is a good idea, I will integrate it with [Folder Dropdown](https://github.com/rocksdanister/lively/wiki/Web-Guide-IV-:-Interaction#folder-dropdown) API.
Unfortunately this won't work without modifying the Javascript code of the wallpaper also.
